gradle.properties

```diff
@@ -7,22 +7,22 @@ javaxPersistenceVersion = 1.0.2.Final
 javaxELAPIVersion = 5.0.1
 javaxELVersion = 4.0.2
 slf4jVersion = 2.0.6
-guavaVersion = 31.1-jre
-undertowVersion = 2.3.10.Final
+guavaVersion = 33.3.1-jre
+undertowVersion = 2.3.17.Final
 resteasyVersion = 6.2.7.Final
 jbossLoggingAnnotationsVersion = 2.0.2.Final
-keycloakVersion = 23.0.6
+keycloakVersion = 25.0.3
 classgraphVersion=4.8.149
 jacksonVersion = 2.16.0
-groovyVersion = 3.0.10
+groovyVersion = 4.0.24
 groovySandboxVersion = 4.0.0
-sqliteVersion = 3.34.0
+sqliteVersion = 3.41.2.2
 hibernateVersion = 6.3.1.Final
 hibernateValidatorVersion = 8.0.0.Final
 hikaricpVersion = 5.0.1
-postgresJdbcVersion = 42.7.1
+postgresJdbcVersion = 42.7.4
 junitVersion = 4.13.2
-spockVersion = 2.0-groovy-3.0
+spockVersion = 2.4-M4-groovy-4.0
```

Comment:
Is there a reason for using a milestone release instead of the latest stable release (2.3-groovy-4.0)?